packages/yoga/src/Text/web/__snapshots__/Text.test.jsx.snap

```diff
@@ -295,32 +370,32 @@ exports[`<Text /> Snapshots should match snapshot with Text light 1`] = `
   padding: 0;
   font-size: 48px;
   line-height: 56px;
-  font-weight: 500;
   font-weight: 300;
-  font-family: Rubik;
   color: #231B22;
+  font-family: Rubik;
+  font-weight: 500;
```

Comment:
This and the others components with the `light` prop should keep the weight of 300. (Order matters)